guides/keras_cv/classification_with_keras_cv.py

```diff
@@ -0,0 +1,237 @@
+- Fine tuning a pretrained backbone
+
+We use Professor Keras, the official Keras mascot, as a
+visual reference for the complexity of the material:
+
+![](https://storage.googleapis.com/keras-nlp/getting_started_guide/prof_keras_evolution.png)
+
+Please note that due to classification being a pretty simple use case, this guide only covers beginner and intermediate work flows.
+Advanced and expert workflows may be found in the [other KerasCV guides](https://keras.io/guides/keras_cv/)!
+"""
+
+"""shell
+!pip install -q --upgrade git+https://github.com/keras-team/keras-cv.git tensorflow
+"""
+
+import json
+import keras_cv
+import tensorflow as tf
+import tensorflow_datasets as tfds
+import keras
+import numpy as np
+
+"""
+## Inference with a pretrained classifier
+
+![](https://storage.googleapis.com/keras-nlp/getting_started_guide/prof_keras_beginner.png)
+
+Let's get started with the simples KerasCV API: a pretrained classifier.
+In this example we will build a Dogs vs Cats classifier using a model that was
+pretrained on the ImageNet dataset.
+
+The highest level module in KerasCV is a *task*. A *task* is a `keras.Model`
+consisting of a (generally pretrained) backbone model and task-specific layers.
+Here's an example using keras_cv.models.ImageClassifier with a ResNet50V2
+Backbone.
+
+
+"""
+
+import json
+import keras_cv
+import tensorflow as tf
+import tensorflow_datasets as tfds
+import keras
+import numpy as np
+
+# TODO(lukewood): replace with ImageClassifier.from_preset()
+classifier = keras_cv.models.DenseNet121(
+    weights='imagenet',
+    include_top=True,
+    include_rescaling=True,
+    num_classes=1000
+)
+
+filepath = tf.keras.utils.get_file(origin="https://i.imgur.com/9i63gLN.jpg")
+image = keras.utils.load_img(filepath)
+
+"""
+![Picture of a cat with a velvet blanket](https://i.imgur.com/9i63gLN.jpg)
+"""
+
+image = np.array(image)
+
+predictions = classifier.predict(image[None, ...])
+top_classes = predictions[0].argsort(axis=-1)
+
+# Subset of imagenet classes
+classes = {
+    281: 'tabby, tabby cat',
+    885: 'velvet'
+}
+top_two = [classes[i] for i in top_classes[-2:]]
+print("Top two classes are:", top_two)
+
+"""Great!  Both of these are correct!  But what if you don't care about the
+velvet blanket, and instead only want to know if a cat is in the image or not?
+This can be solved using fine tuning.
+
+# Fine tuning a pretrained classifier
+
+![](https://storage.googleapis.com/keras-nlp/getting_started_guide/prof_keras_intermediate.png)
+
+When labeled images specific to our task are available, fine-tuning a custom
+classifier can improve performance. If we want to train a Cats VS Dogs
+Classifier, using explicitly labeled Cat vs Dog data should perform better than
+the generic classifier data! And for many tasks, no relevant pretrained model
+will be available (e.g., categorizing images specific to your application).
+
+The biggest difficulty when finetuning a KerasCV model is loading and augmenting
```

Comment:
Data augmentation should be a level two section at least. This doc could use more sections.